Code/Editor/PluginManager.cpp

```diff
@@ -269,13 +268,13 @@ void CPluginManager::RegisterPlugin(QLibrary* dllHandle, IPlugin* pPlugin)
 
 IPlugin* CPluginManager::GetPluginByGUID(const char* pGUID)
 {
-    for (auto it = m_plugins.begin(); it != m_plugins.end(); ++it)
+    for (const SPluginEntry& plugEntry : m_plugins)
```

Comment:
Nit: "pluginEntry" or "plugin" would make a bit more sense in this context than "plugEntry".